### PR TITLE
Fix create issue / pull failed when creating 1 issue + 1 pull

### PR DESCRIPTION
**Pull request by linquize**
Fix create issue / pull failed when creating 1 issue + 1 pull, because of index collision

### CODE REVIEW

**Comment by lunny** [comment]
I prefer to set [repo_id, index, is_pull] as an unique instead of [repo_id, index]. So that, issues and prs have different index.

**Comment by linquize** [comment]
@lunny But the problem is that it is difficult for user to reference an issue from a pull / a pull from an issue. notation like `#1`, `#2`
I think a single index for both issues and prs is better.

**Comment by lunny** [comment]
OKay. You are right. Let me merge it.

**Comment by tboerger** [comment]
Just as a side note, i like the single idea approach as well even if gitlab handles it different with #123 issue ref and !123 pull request ref.

**Comment by lunny** [comment]
So now, we are github style on this case.

File: routers/install.go [hunk 1]
```diff
@@ -34,7 +34,8 @@ import (
 )
 
 const (
-	INSTALL base.TplName = "install"
+	// tplInstall template for installation page
```
Comment (bkcsoft) [inline routers/install.go:37]: no need for a comment here since it isn't exported :slightly_smiling_face:

**Pull request merged**